### ISSUE TITLE
webserver still active in latest build?

### ISSUE THREAD

**Issue by leggewie** [freelan-developers/freelan#150]
I compiled master and then issued a "freelan --help | head -n 28".  I got
````
Generic options:
  -h [ --help ]                   Produce help message.
  -v [ --version ]                Get the program version.
  -d [ --debug ]                  Enables debug output.
  -t [ --threads ] arg (=0)       The number of threads to use.
  -c [ --configuration_file ] arg The configuration file to use.

Configuration:

FreeLAN Server options:
  --server.enabled arg (=no)            Whether to enable the server mechanism.
  --server.listen_on arg (=0.0.0.0:443) The endpoint to listen on.
  --server.protocol arg (=https)        The protocol to use for clients to 
                                        contact the server.
  --server.server_certificate_file arg (="")
                                        The server certificate file.
  --server.server_private_key_file arg (="")
                                        The server private key file.
  --server.certification_authority_certificate_file arg (="")
                                        The certification authority certificate
                                        file.
  --server.certification_authority_private_key_file arg (="")
                                        The certification authority private key
                                        file.
  --server.authentication_script arg (="")
                                        The authentication script to use.
````

I thought the web server was disabled in the latest builds?

**Comment by s-vincent** [comment]
Yes it is! If you launch it you will have:
 `[WARNING] Web server support is not compiled in this version.`

But I just forgot to conditionnaly remove the help options. I will do it today.

**Comment by s-vincent** [comment]
Fixed in 289e5ececeb8e259cd4ae5dc525ac837913d867a.

**Comment by s-vincent** [comment]
Previous commit comes with a regression. It failed reading classic configuration file... I am on it.

**Comment by s-vincent** [comment]
Fixed in 09a28364ccd3a6779dd8b4a9a43888ed8738399d.
